OPERATING.md

```diff
@@ -0,0 +1,151 @@
+# Operating
+
+This document contains requirements and tips for operating the adaptor in a production environment.
+
+# AMQP Message Broker Requirements
+
+* The broker must be configured with a limited number of retries and deadletter queues
```

Comment:
I think this should be separate words or hyphenated
`dead letter queues`, or `dead-letter queues`.

Reply:
I'll go with hyphenated.